Resources/Prototypes/Floof/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -0,0 +1,308 @@
+- type: entity
+  name: basic
+  id: MobSpaceFriend
+  parent: MobXenoRounyNPC
+  description: It looks friendly. Why don't you give it a hug?
+  components:
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: MovementAlwaysTouching
+  - type: MeleeWeapon
+    soundHit:
+        path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
+    angle: 0
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Blunt: 5
+        Slash: 5
+  - type: Puller
+    needsHands: false
+  - type: GhostTakeoverAvailable
+
+- type: entity
+  name: Neutral Rouny
+  id: MobXenoNeutralRouny
+  parent: MobSpaceFriend
+  description: They mostly come at night. Mostly.
+  components:
+  - type: NpcFactionMember
+    factions:
+    - Passive
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon
+
+- type: entity
+  name: Neutral Praetorian
+  id: MobXenoNeutralPraetorian
+  parent: MobXenoPraetorianNPC
+  description: They mostly come at night. Mostly.
+  components:
+  - type: NpcFactionMember
+    factions:
+    - Passive
+  - type: Puller
+    needsHands: false
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon
+
+- type: entity
+  name: Neutral Drone
+  id: MobXenoNeutralDrone
+  parent: MobXenoDroneNPC
+  description: They mostly come at night. Mostly.
+  components:
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon
+
+- type: entity
+  name: Neutral Ravager
+  id: MobXenoNeutralRavager
+  parent: MobXenoRavagerNPC
+  description: They mostly come at night. Mostly.
+  components:
+  - type: NpcFactionMember
+    factions:
+    - Passive
+  - type: LanguageKnowledge
+    speaks:
+    - Xeno
+    understands:
+    - Xeno
+    - GalacticCommon
+
+- type: inventoryTemplate
+  id: friendxeno
+  slots:
+  - name: id
+    slotTexture: id
+    slotFlags: IDCARD
+    slotGroup: SecondHotbar
+    stripTime: 6
+    uiWindowPos: 2,1
+    strippingWindowPos: 2,4
+    displayName: ID
+  - name: pocket1
+    slotTexture: pocket
+    slotFlags: POCKET
+    slotGroup: MainHotbar
+    stripTime: 3
+    uiWindowPos: 0,3
+    strippingWindowPos: 0,4
+    displayName: Pocket 1
+    stripHidden: false
+
+- type: entity
+  name: Friend-Shaped
+  parent: MobSpaceFriend
+  id: MobXenoFriendShaped
+  description: A very clearly friend-shaped Xeno.
+  components:
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/Xenos/rouny.rsi
+    scale: 0.7, 0.7

```

Comment:
If parent already has rouny sprite, this is not needed.

Reply:
I specified it so I could choose a rescale, I am uncertain how to give the scale change for friend-shaped without this.